Model/ResourceModel/Notification.php

```diff
@@ -49,4 +52,27 @@ public function getNotification($pspReference, $eventCode, $success, $originalRe
 
         return $this->getConnection()->fetchAll($select);
     }
+
+    /**
+     * Deletes the rows corresponding to the given `entity_id`s
+     *
+     * @param array $entityIds
+     * @return void
+     * @throws LocalizedException
+     */
+    public function deleteByIds(array $entityIds): void
+    {
+        if (empty($entityIds)) {
+            return;
+        }
+
+        $tableName = $this->getMainTable();
+
+        $connection = $this->getConnection();
+        $select = $connection->select()
+            ->from(['notification' => $tableName])
```

Comment:
Is `notification` a constant in another class? If so we could use it here to avoid using a string

Reply:
Thank you for your comment @acampos1916! No, `notification` is not a constant and reflects to the alias of the table name. It can be any string as long as the same value is used while building DELETE query `$connection->query($select->deleteFromSelect('notification'));`

I will replace it with a constant to overcome the confusion.

Reply:
Please see the related commit [here](https://github.com/Adyen/adyen-magento2/pull/2837/commits/e8d5fd1552725689a59127bae20b0b7cee71e350)